guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>The automatic lr_finder will return a value <s>taking</s> <u>that takes </u>a conservative estimate of the learning rate. <u>While the conservative estimate generated by lr_finder is reliable</u>, an expert <s>can</s> interpret the graph more appropriately and find a better learning rate for the final model training. In the figure below, the suggested learning rate by the <code>lr_find()</code> method <s>was</s> <u>is </u>around 0.0002754.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>